### ISSUE TITLE
Chapter MOU, delete/void MOU functionaity

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#4726]
If an MOU has been sent (and not signed yet) and for some reason it needs to be deleted or re-sent; we need a way to delete it and void it on the DocuSign side and delete our record locally.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/4726?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by dboyer** [comment]
Tested on QA and found:

- Void MOU button did not appear until the MOU had been sent
- I clicked the Void button and saw a message at the top of the page indicating it was voided
- As the intended signer, I received an email saying the doc was voided
- However, when I tried to access the MOU in the original email I received with instructions to sign it, I was still able to click through and sign that document. The [admin page for the chapter says it was signed](https://technovation-qa.herokuapp.com/admin/chapters/133#/).

What's interesting is that when I sent the legal contact the MOU, the legal contact received two emails. After I voided the MOU, the link in the first email didn't work anymore but the link in the second email did. I'll forward those emails to you @shaun-technovation
